modules/preview-react/radio/lib/RadioText.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import {createSubcomponent, ExtractProps} from '@workday/canvas-kit-react/common';
+import {inputColors, colors} from '@workday/canvas-kit-react/tokens';
+import {useRadioModel} from './hooks/useRadioModel';
+import {RadioLabelContext} from './RadioLabel';
+import {Text} from '@workday/canvas-kit-react/text';
+
+export const RadioText = createSubcomponent('span')({
+  displayName: 'RadioButton.Text',
+  modelHook: useRadioModel,
+})(({children, ...elemProps}: ExtractProps<typeof Text>, Element) => {
+  const {disabled, variant} = React.useContext(RadioLabelContext);
+
+  return (
+    <Text
+      as={Element}
+      style={{
+        cursor: disabled ? undefined : 'pointer',
+        opacity: disabled ? (variant === 'inverse' ? '.4' : '1') : undefined,
```

Comment:
```suggestion
        opacity: disabled && inversed ? '.4' : '1'
```

Reply:
I feel it's ok to keep default value as 1, because it's default value for css as well